backend/app.ts

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import cors from 'cors';
+
+const app = express();
+
+app.use(express.json());
+
+app.use(
+    cors({
+        origin: 'http://localhost:3000',
```

Comment:
choose a different port, as this is conventionally what frontend runs on when we are operating locally